requirements.txt

```diff
@@ -1,18 +1,55 @@
-Django~=1.11
```

Comment:
@StrangeStuff Tell me, did you generate this from a pip list > requirements.txt? IMO this list should be reduced to it's simplest version i.e only the main dependency you added like 'redis==3.2.1' not more. Anyway to modify this ?

Reply:
Ok, I got it ! I had done a pip freeze and it is not very clean. For th most recent commits, I took the most recent requirements.txt from the master branch and I have added only the 3 needed libraries, redis, celery and django-celery-beat.

Reply:
:+1: